Create 2021-01-19-changes-to-rustdoc-team.md (#761)
rust-lang/blog.rust-lang.org: posts/inside-rust/2021-01-19-changes-to-rustdoc-team.md

---
layout: post

> title: "Changes to Rust rustdoc team"

Reviewer: I think 'Rust' is assumed, since this is the inside-rust blog :P
```suggestion
title: "Changes to the Rustdoc team"
```

Author: We're never too sure!

author: Guillaume Gomez
description: "leadership and membership additions"
team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
---

Recently, there have been a lot of improvements in rustdoc. It was possible thanks to our new contributors. In light of these recent contributions, a few changes were made in the rustdoc team.

## Adding a new team leader

> [Joshua](https://github.com/jyn514) has been helping me **a lot** on the leading parts by tagging issues and mentoring newcomers. But at the same time, he initiated great changes in the rustdoc codebase that led to huge performance improvements, he finished the "intra doc link" features which is now stable after years of instability. He also did multiple additional things but let's stop the listing here. :)

Reviewer: It would be nice to link to https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html here :)

Author: Oh good idea!

> Overall, he's been a monstruous help and making him co-lead is just the officialization of everything's he's been doing since he joined the rustdoc team (and even before!) and I'm glad to have him on the team and as co-lead.

Reviewer: Heh, I hope my help is not "extremely and dauntingly large" :laughing:
```suggestion
Overall, he's been a enormous help and making him co-lead is just making offical everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.
```

> Overall, he's been a enormous help and making him co-lead is just making offical everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.

Reviewer 2: ```suggestion
Overall, he's been an enormous help and making him co-lead is just making offical everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.
```

> Overall, he's been an enormous help and making him co-lead is just making offical everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.

Reviewer 3: Tiny typo

```suggestion
Overall, he's been an enormous help and making him co-lead is just making official everything's he's been doing since he joined the rustdoc team (and even before!). I'm glad to have him on the team and as co-lead.
```

Reviewer: Thanks, I openedhttps://github.com/rust-lang/blog.rust-lang.org/pull/765

## Membership Changes

> In addition to the new co-leadership nomination, we also have some new rustdoc team members to announce: [camelid] and [Nemo157].

Reviewer: These are usernames, so I would bold them. (@camelid , @Nemo157 let me know if you'd prefer to use real names instead)
```suggestion
In addition to the new co-leadership nomination, we also have some new rustdoc team members to announce: [**@camelid**][camelid] and [**@Nemo157**][Nemo157].
```

Author: Fine by me. BTW, do you prefer me using your username or your name?

Reviewer: I prefer using my real name, thanks.

> [camelid] did a huge number of contributions on rustdoc, helping on the cleanup and fixing bugs.

Reviewer: ```suggestion
**@camelid** did a huge number of contributions on rustdoc, helping on the cleanup and fixing bugs.
```

> [Nemo157] helped a lot on the "front-end" part of rustdoc (understand: HTML/CSS/JS), not only reviewing but also fixing bugs and improving the current code.

Reviewer: ```suggestion
**@Nemo157** helped a lot on the "front-end" part of rustdoc (i.e. HTML/CSS/JS), not only reviewing but also fixing bugs and improving the current code.
```
